src/status_im/chat/models/input.cljs

```diff
@@ -66,6 +66,11 @@
   (let [current-chat-id (or chat-id (:current-chat-id db))]
     {:db (assoc-in db [:chat/inputs current-chat-id :audio] audio)}))
 
+(rf/defn set-recording
+  {:events [:chat.ui/set-recording?]}
```

Comment:
I'd suggest naming the event `:chat.ui/set-recording`, without the question mark, or `:chat.ui/set-recording-flag`.
For me the question mark in the name of a setter is a bit disorienting since I read it with a questioning intonation)

Reply:
Thanks for the suggestion, done